src/org/traccar/database/PermissionsManager.java

```diff
@@ -176,9 +177,9 @@ public void checkDeviceLimit(long userId) throws SecurityException, SQLException
         if (deviceLimit != -1) {
             int deviceCount = 0;
             if (isManager(userId)) {
-                deviceCount = Context.getDeviceManager().getManagedDevices(userId).size();
+                deviceCount = Context.getDeviceManager().getManagedItems(userId).size();
             } else {
-                deviceCount = getDevicePermissions(userId).size();
+                deviceCount = Context.getDeviceManager().getUserItems(userId).size();
```

Comment:
Basically nothing changed here, just a couple function calls, but it looks more consistent.